modules/sh-camera/backend/modules/sh_camera/README.md

```diff
@@ -0,0 +1,7 @@
+## Requirements
+
+-   pip install djangorestframework
```

Comment:
No need to install `djangorestframework`, its already included in the scaffold:
https://github.com/crowdbotics/django-scaffold/blob/develop/%7B%7Bcookiecutter.project_slug%7D%7D/Pipfile#L20